apps/site/pages/en/index.mdx

```diff
@@ -14,29 +14,25 @@ layout: home
 
   </div>
 
-  <div>
-    <WithNodeRelease status={['LTS']}>
-      {({ release }) => (
-        <>
-          <DownloadButton release={release}>Download Node.js (LTS)</DownloadButton>
-          <small>
-            Downloads Node.js <b>{release.versionWithPrefix}</b>
-            <sup title="Downloads a Node.js installer for your current platform">1</sup> with long-term support.
-            Node.js can also be installed via <Link href="/download">version managers</Link>.
-          </small>
-        </>
-      )}
-    </WithNodeRelease>
-
-    <WithNodeRelease status={['Current']}>
-      {({ release }) => (
-        <small>
-          Want new features sooner?
-          Get <b>Node.js <DownloadLink release={release}>{release.versionWithPrefix}</DownloadLink></b>
-          <sup title="Downloads a Node.js installer for your current platform">1</sup> instead.
-        </small>
-      )}
-    </WithNodeRelease>
+  <div className="flex gap-4">
+
+    <div className="flex flex-col gap-2">
+      <Button kind="special" className="!hidden dark:!block w-100" href="/download">Install Node.js</Button>
+
+      <Button kind="primary" className="!block dark:!hidden w-100" href="/download">Install Node.js</Button>
+
+      <Button kind="secondary" className="!hidden dark:!block w-100" href="https://www.herodevs.com/support/node-nes">
```

Comment:
This will need UTM params I imagine?

Reply:
IDK what the campaign link should be 🤷

Reply:
@joeeames what UTM params would you like for the homepage?

Reply:
Bump, @joeeames

Reply:
I'm not going to keep this PR blocked because @joeeames isn't able to answer this question in a timely manner. UTM query params can always be added later.